spec/models/conference_spec.rb

```diff
@@ -41,12 +41,21 @@
       end
     end
   end
+  
+  describe '#dates' do
+    subject { FactoryGirl.build_stubbed(:conference) }
+    
+    it 'has a date range' do
+      expect(subject.date_range).to be_present
```

Comment:
Hm, I'm not a huge fan of `be_present` or `be` alone, I think it does not tell anything while still being as slow as most other expectations... How about at least do a
```ruby
be_a(String)
```
?